tests/Feature/api/v1/Room/RoomTest.php

```diff
@@ -346,27 +350,37 @@ public function testRoomSearch()
         // Testing without query
         $this->actingAs($this->user)->getJson(route('api.v1.rooms.index').'?filter=own')
             ->assertOk()
-            ->assertJsonFragment(['id'=>$room->id]);
+            ->assertJsonFragment(['id'=>$room->id])
+            ->assertJsonPath('meta.total', 1)
+            ->assertJsonPath('meta.total_no_filter', 1);
 
         // Testing with empty query
         $this->actingAs($this->user)->getJson(route('api.v1.rooms.index').'?filter=own&search=')
             ->assertOk()
-            ->assertJsonFragment(['id'=>$room->id]);
+            ->assertJsonFragment(['id'=>$room->id])
+            ->assertJsonPath('meta.total', 1)
+            ->assertJsonPath('meta.total_no_filter', 1);
 
         // Testing with fragment of the room name
         $this->actingAs($this->user)->getJson(route('api.v1.rooms.index').'?filter=own&search=One')
             ->assertOk()
-            ->assertJsonFragment(['id'=>$room->id]);
+            ->assertJsonFragment(['id'=>$room->id])
+            ->assertJsonPath('meta.total', 1)
+            ->assertJsonPath('meta.total_no_filter', 1);
 
         // Testing with full name
         $this->actingAs($this->user)->getJson(route('api.v1.rooms.index').'?filter=own&search=Meeting One')
             ->assertOk()
-            ->assertJsonFragment(['id'=>$room->id]);
+            ->assertJsonFragment(['id'=>$room->id])
+            ->assertJsonPath('meta.total', 1)
+            ->assertJsonPath('meta.total_no_filter', 1);
 
         // Testing with invalid name
         $this->actingAs($this->user)->getJson(route('api.v1.rooms.index').'?filter=own&search=Meeting Two')
             ->assertOk()
-            ->assertJsonMissing(['id'=>$room->id]);
+            ->assertJsonMissing(['id'=>$room->id])
+            ->assertJsonPath('meta.total', 0)
+            ->assertJsonPath('meta.total_no_filter', 1);
```

Comment:
It would be nice to have a test where no filter is set, to ensure the meta is not set!

Reply:
As there is no assertJsonPathMissing I added a count test of the meta object attributes to the filter and non-filter request